portal/src/texts/universell-utforming/Roller.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Roller
+path: /universell-utforming/roller
+order: 7
+---
+
+# Universell utforming og roller
+Universell utforming er viktig igjennom hele organisasjonen, det er ikke en enkelt fagdisiplin eller gruppe som er alene ansvarlig. Det kan være en fare med kvalitetskrav som universell utforming som krever arbeid eller endring i arbeidet fra hele organisasjonen at det kan falle mellom to stoler eller at fagdisipliner dytter ansvaret mellom seg. For å unngå at det skjer er det viktig at alle har forståelse for hva som forventes av sin rolle og hva en skal forvente av andre. Her er rollene delt opp i større grupper som vil inneholde undergrupper. Det er opp til hvert enkelt team å sørge for at man dekker alle områdene det forventes, her vil man også kunne dele på tvers av inndelingene her og tradisjonelle ansvarsfordelinger. Det er teamet som helhet som er ansvarlig for kvaliteten på produktet sitt.
+
+- [Alle](#alle)
+- [UX](#ux)
+- [Utvikling](#utvikling)
+- [Test](#test)
+- [Produkt](#produkt)
+- [Support](#support)
+
+## Alle
+Det forventes at alle har en grunnlegende forståelse for hva universell utforming er og hva det betyr for oss som organisasjon. Det forventes også at det er fokus på temaet, ikke fordi det er lovpålagt, men fordi det er det gjør hos Fremtind. Å bygge kvalitetskravene inn i prosessen og produktene våre tar tid, og det viktig at alle er med å drar i riktig retning og forstår at dette tar litt tid å få til riktig.
+
+Det forventes:
+- Generell forståelse av hva universell utforming er
+- Forståelse av hva som forventes av de ulike rollene
+- Bidra til å sette fokus på universell utforming
+- Hjelpe hverandre til å øke forstålse og kunnskap
+
+## UX
+Designere og UXere er "førsteforsvaret" for universell utforming. Det er helt kritisk for å kunne levere en god opplevelse for brukerene vår at universell utforming er med fra første stund, det kan ikke være noe man legger på etter at man har designet løsningen. 
```

Comment:
Er noe feil med denne setningen:

 "Det er helt kritisk for å kunne levere en god opplevelse for brukerene vår at universell utforming er med fra første stund, det kan ikke være noe man legger på etter at man har designet løsningen. " Er noe feil med denne setningen

Kanskje noe sånt:
"Det er helt avgjørende at universell utforming er med fra fra start til slutt når man lager et brukergrensesnitt. Det bør inngå som en del av planleggingen før man setter i gang med å lage en designskisse."